crawl-ref/source/tileweb.cc

```diff
@@ -1621,11 +1621,13 @@ void TilesFramework::_send_cell(const coord_def &gc,
             write_tileidx(next_pc.cloud);
         }
 
-        if (next_pc.is_bloody != current_pc.is_bloody)
-            json_write_bool("bloody", next_pc.is_bloody);
+        if (Options.show_blood) {
```

Comment:
maybe better just to send `false`. Also, `old_blood` is not used if `bloody` is false, so can be ignored here.